Automattic/jetpack-vscode
# Tracks: Enable tracks when the rsync command is called

This PR will fire a tracks event when the rsync command is used. 

The values are temporary for now until we determine the best naming convention. I'm thinking `jetpack_vscode_command` and then we can pass it the property for the specific command being run, i.e. `rsync`.

I believe we can also remove the `_ul` property? 

Does the User-Agent `Jetpack VSCode Extension` make sense ?

- [x] todo: add a setting so folks can enable or disable tracking.

To test the PR as is:

- Pull branch down
- npm run compile
- Open extension.ts and hit F5
- In the new vscode window, choose the project you want to test with, like the jetpack monorepo. 
- ctrl + p -> choose the Jetpack: rsync option and complete the command
- In about 5 minutes, check the tracks live view to make sure it fired. 
- Search for the analytic extension and disable/enable the toggle to make sure the event isn't fired when it's disabled.
